sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -1623,7 +1674,8 @@ pi_result piextDeviceSelectBinary(pi_device, pi_device_binary *,
 pi_result piextUSMEnqueuePrefetch(pi_queue, const void *, size_t,
                                   pi_usm_migration_flags, pi_uint32,
                                   const pi_event *, pi_event *) {
-  DIE_NO_IMPLEMENTATION;
+  // NOP for prefetch
```

Comment:
Please explain in the comment why NOP for prefetch is safe.

Reply:
I'll replace it with TODO like above ones.